ecflow/ecf/scripts/gdas/atmos/post/jgdas_atmos_post_f009.ecf

```diff
@@ -3,8 +3,8 @@
 #BSUB -P %PROJ%
 #BSUB -J %E%gdas_atmos_post_%FHR%_%CYC%
 #BSUB -o %COM%/output/%ENVIR%/today/gdas_atmos_post_%FHR%_%CYC%.o%J
-#BSUB -W 0:10
-#BSUB -n 56
+#BSUB -W 2:00
```

Comment:
2 hours is an awful long time to reserve a job for a single forecast hour to be postprocessed.
The scheduler should not submit this job until the file is in place for it to work on it.